src/mpid/ch4/netmod/ofi/ofi_comm.c

```diff
@@ -18,7 +18,7 @@ static int update_multi_nic_hints(MPIR_Comm * comm)
             int was_enabled_striping = MPIDI_OFI_COMM(comm).enable_striping;
 
             /* Check if we should use striping */
-            if (comm->hints[MPIR_COMM_HINT_MULTI_NIC_PREF_NIC] == 1) {
+            if (comm->hints[MPIR_COMM_HINT_MULTI_NIC_PREF_NIC] != -1) {
```

Comment:
The commit message ("ch4/ofi: correct detection of nic hints under multi-nic") was wrong. It is "fix checking PREF_NIC hints". 

How about define a macros at the top of this file, such as:
```
    #define HAS_PREF_NIC(comm) comm->hints[MPIR_COMM_HINT_MULTI_NIC_PREF_NIC] != -1
```
It should help reading and prevent future bug.
```

Reply:
Such a wrapper should be better.

Reply:
Done